### DISCUSSION TITLE
Azure Storage Dependency Updates

### DISCUSSION THREAD

**Discussion by MauriceReeves-usds** [CDCgov/prime-reportstream#1830] [General]
So I spent some time looking at the Azure Storage Queue and Azure Storage Blob dependency updates, and it turns out that the problem with the updates was they were incompatible with the azurite image we were using. It's very frustrating because there was no good documentation that told us that. Once I bumped the azurite image in Docker to the latest version those two dependencies worked great. I've checked in that change. If you see an issue with `x-ms-version` being in the wrong format, it is likely a mismatch between the Azure library and the azurite docker image.

**Comment by td-usds** [comment]
As per @mauricereeves 
>  If you see an issue with x-ms-version being in the wrong format, it is likely a mismatch between the Azure library and the azurite docker image.

This would manifest itself in the logs produces by the Azurite container
